data/inputs/critProperties.xml

```diff
@@ -76,7 +76,7 @@
     <Pc value="7.39e+06"/>
     <Vc value="0.0948"/>
     <Zc value="0.275"/>
-    <omega value="0.239"/>
+    <omega value="0.228"/>
```

Comment:
Please add a reference to a source here.

Reply:
I have added the reference for this value.